src/qml/pages/EditProfilePage.qml

```diff
@@ -0,0 +1,188 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
+
+import "../theme"
+import "../components"
+
+//Q_PROPERTY(QString userId READ userId WRITE setUserId NOTIFY userIdChanged)
+//Q_PROPERTY(QString phoneNumber READ phoneNumber WRITE setPhoneNumber NOTIFY phoneNumberChanged)
+//Q_PROPERTY(QString email READ email WRITE setEmail NOTIFY emailChanged)
+//Q_PROPERTY(bool isPhoneNumberConfirmed READ isPhoneNumberConfirmed WRITE setIsPhoneNumberConfirmed NOTIFY isPhoneNumberConfirmedChanged)
+//Q_PROPERTY(bool isEmailConfirmed READ isEmailConfirmed WRITE setIsEmailConfirmed NOTIFY isEmailConfirmedChanged)
+//Q_PROPERTY(QUrl avatarUrl READ avatarUrl WRITE setAvatarUrl NOTIFY avatarUrlChanged)
+
+OperationPage {
+    id: editProfilePage
+    appState: app.stateManager.editProfileState
+    property bool isPhoneConfirmed: appState.isPhoneNumberConfirmed
+    property bool isEmailConfirmed: appState.isEmailConfirmed
+
+    loadingText: qsTr("Opening profile page...")
+
+    header: Header {
+        showBackButton: !confirmForm.isLoading
+        title: editProfilePage.state === "standard" ? qsTr("Profile") : qsTr("Confirm")
+    }
+
+    state: "standard"
+    states: [
+        State {
+            name: "standard"
+            PropertyChanges {
+                target: profileForm
+                opacity: 1
+                enabled: true
+                scale: 1
+            }
+            PropertyChanges {
+                target: confirmForm
+                opacity: 0
+                enabled: false
+                scale: 0.8
+            }
+        },
+        State {
+            name: "phone confirmation"
+            PropertyChanges {
+                target: profileForm
+                opacity: 0
+                enabled: false
+                scale: 1.2
+            }
+            PropertyChanges {
+                target: confirmForm
+                opacity: 1
+                enabled: true
+                scale: 1
+            }
+        },
+        State {
+            name: "email confirmation"
+            PropertyChanges {
+                target: profileForm
+                opacity: 0
+                enabled: false
+                scale: 1.2
+            }
+            PropertyChanges {
+                target: confirmForm
+                opacity: 1
+                enabled: true
+                scale: 1
+            }
+        }
+    ]
+
+    transitions: Transition {
+        NumberAnimation {properties: "opacity, scale, enabled"; duration: Theme.animationDuration; easing.type: Easing.InExpo}
+    }
+
```

Comment:
Not a big deal, but stop adding 2/3/more empty lines in a row. One is enough, isn't it?

Reply:
I showed the intermediate version of the code not for you to count the lines, but to see the logic.